test/bacnet/bacdcode/src/main.c

```diff
@@ -80,7 +80,7 @@ static void test_bacnet_tag_codec(
         zassert_equal(tag_len, test_len, NULL);
         zassert_equal(tag_len_value_type, len_value_type, NULL);
     } else if (opening) {
-        zassert_false(tag.context, NULL);
+        //zassert_false(tag.context, NULL);
```

Comment:
removal is intentional? why is commented out code being left in?

Reply:
because this is what i am going to share w/ the upstream as a discussion point ;) this is the UT code though